src/EFCore.PG/Query/Internal/NpgsqlSqlTranslatingExpressionVisitor.cs

```diff
@@ -31,22 +31,25 @@ public class NpgsqlSqlTranslatingExpressionVisitor : RelationalSqlTranslatingExp
                 .GetRuntimeMethod(nameof(DbFunctionsExtensions.Like), new[] { typeof(DbFunctions), typeof(string), typeof(string) });
 
         // ReSharper disable once InconsistentNaming
-        static readonly MethodInfo ILike2MethodInfo =
-            typeof(NpgsqlDbFunctionsExtensions)
+        static readonly MethodInfo ILike2MethodInfo
+            = typeof(NpgsqlDbFunctionsExtensions)
```

Comment:
Any reason to change the style? As I remember we leave assignment on the first line everywhere because of indentation. Right now, instead of 4 characters offset, for such lines it's 6.

Reply:
Mainly for consistency with expression-bodied methods, where we put the `=>` on the next line.

Reply:
I think putting `=>` on the first line is better too for the same reason plus for sharing styling with .NET runtime.

Reply:
The EF Core standard (and this is an EF Core provider) is to put it on the next line (usually... sometimes a bit inconsistent :))

Why is a 4 char offset better than 6 char?

Reply:
Because of alignment with non-expression bodied methods. For example, prefixed commit messages (don't remember how that convention is called) is bad too due to how people read text. Monospace fonts were made to help human beings to scan files vertically without jumps vertically, but inconsistent indentation breaks that achievement.

Reply:
I really don't see any reason for expression-bodied methods to start at the same offset as non-expression-bodied methods... They're different methods...

One good argument for putting it on the next line, is that it makes it immediately clear if it's a field initialization or a property with a getter only. This crucial difference can easily get lost if it's the arrow or assignment is far to the right on the first line.

Reply:
It's not a hill I want die for. I said my opinion, but leaving it up to you.